### ISSUE TITLE
crictl is required by the container runtime: executable file not found in %PATH%

### ISSUE THREAD

**Issue by mloskot** [kubernetes-sigs/sig-windows-dev-tools#248]
```
==> winw1: Running provisioner: shell...
    winw1: Running: sync/shared/kubejoin.ps1 as C:\tmp\vagrant-shell.ps1
    winw1: [preflight] Running pre-flight checks
    winw1: W0417 03:53:08.913604    4060 checks.go:1019] [preflight] WARNING: Couldn't create the interface used for talking to the container runtime: 
       crictl is required by the container runtime: executable file not found in %PATH%
```

AFAICT, there is no `crictl.exe` in the `PATH`

1. `vagrant ssh winw1`
2. List `C:\Program Files\containerd` content - location added to `PATH`:
      ```
      PS C:\> ls '.\Program Files\containerd\'
      
          Directory: C:\Program Files\containerd
      
      Mode                LastWriteTime         Length Name
      ----                -------------         ------ ----
      -a----        4/17/2023   3:50 AM           6328 config.toml
      -a----         1/5/2023  10:42 AM       16999424 containerd-shim-runhcs-v1.exe
      -a----         1/5/2023  10:42 AM       16859648 containerd-stress.exe
      -a----         1/5/2023  10:42 AM       35206144 containerd.exe
      -a----        4/17/2023   3:50 AM       32503306 containerd.tar.gz
      -a----        1/21/2022   3:44 AM       19034557 crictl-v1.22.0-windows-amd64.tar.gz
      -a----         1/5/2023  10:42 AM       17571840 ctr.exe
      ```

A while ago there was this patch merged to `sync/linux/controlplane.sh`

- #145

with `mv "C:\Users\vagrant\crictl.exe" "C:\Program Files\containerd\"`, but it was lost later in 

- #160

The `crictl.exe` remains in the `$env:HOME`

```
PS C:\> dir .\Users\vagrant\crictl.exe

    Directory: C:\Users\vagrant

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         8/5/2021   5:33 AM       46650880 crictl.exe
```

**Comment by mloskot** [comment]
I fixed this issue in #254 as I needed it while working on `make.ps1` addition.

**Comment by mloskot** [comment]
> I fixed this issue in #254 as I needed it while working on `make.ps1` addition.

I submitted the fix in separate PR #256

**Comment by mloskot** [comment]
The fix has been merged
- #256

Closing this one myself.
